client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1149,6 +1149,10 @@ Undocumented.prototype.readRecommendationsStart = function( query, fn ) {
 	return this.wpcom.req.get( '/read/recommendations/start', query, fn );
 };
 
+Undocumented.prototype.graduateNewReader = function( query, fn ) {
+	return this.wpcom.req.post( '/read/graduate-new-reader', query, fn );
```

Comment:
I've updated to use v1.2 since that's what is used in the example request in the API docs for this endpoint.